javascript/reactjs-todo/.env.example

```diff
@@ -1,7 +1,8 @@
 # APP_URL=$APP_URL # not using this for preview-environment instead, we can use window.location.origin
 AM_URL=$AM_URL
```

Comment:
We might want to change `AM_URL` to `AS_URL`. This might be a lot of work, so we could delay it, but as-is it could be a bit confusing.